1-syntax-basic-constructs/basic.js

```diff
@@ -0,0 +1,2 @@
+let age = window.prompt("Insert your age:");
+alert(age>18?"you're an adult!":"you're not an adult!");
```

Comment:
```suggestion
let age = window.prompt("Insert your age:");
alert(age > 18 ? "you're an adult!" : "you're not an adult!");
```
spaces in between commands are important for readability.